selfdrive/pandad/panda_safety.cc

```diff
@@ -0,0 +1,79 @@
+#include "selfdrive/pandad/pandad.h"
+#include "cereal/messaging/messaging.h"
+#include "common/swaglog.h"
+
+void PandaSafety::configureSafetyMode() {
```

Comment:
this is cleaner than what we had before, but I think it'll be more readable as a single function like "safety_mode_tick"